modules/aioseop_sitemap.php

```diff
@@ -4249,6 +4520,8 @@ public function get_all_post_type_data( $args ) {
 			}
 			$this->excludes = array_merge( $args['exclude'], $exclude_slugs ); // Add excluded slugs and IDs to class var.
 
+			$args = $this->add_tax_query( $args );
```

Comment:
DEV NOTE: Potential conflict with #2317 Same function is being modified.

Reply:
not relevant now as the function is not being modified.